pygmt/tests/test_rose.py

```diff
@@ -0,0 +1,202 @@
+"""
+Tests for rose.
+"""
+import numpy as np
+import pytest
+from pygmt import Figure
+from pygmt.datasets import load_fractures_compilation
+
+
+@pytest.mark.mpl_image_compare
+def test_rose_data_file():
+    """
+    Test supplying data from sample dataset.
+    """
+
+    data = load_fractures_compilation()
+
```

Comment:
I believe the tests would be much faster if load the data into a pytest fixture, e.g., 

```
@pytest.fixture(scope="module")
def data_fractures_compilation():
    """
    Load the sample fractures compilation dataset.
    """
    return load_fractures_compilation()
```
and in any tests that need the data, we can simply use:
```
def test_rose_data_file(data_fractures_compilation):
    fig = Figure()
    fig.rose(
        data=data_fractures_compilation,
        ...
    )
```

Reply:
Ok, done in da6b829fe3679232a8d6978cf9393aef2b7f6db6, and also resolves https://github.com/GenericMappingTools/pygmt/pull/794#discussion_r605323626.